packages/travix-ui-kit/components/calendar/views/days.js

```diff
@@ -209,8 +209,13 @@ class Days extends Component {
     const nextMonth = renderDate.getMonth() === 11 ? 0 : (renderDate.getMonth() + 1);
     const previousMonth = renderDate.getMonth() === 0 ? 11 : (renderDate.getMonth() - 1);
 
-    const isPreviousMonthDisabled = minDate && (minDate.getMonth() >= renderDate.getMonth());
-    const isNextMonthDisabled = maxDate && (maxDate.getMonth() <= renderDate.getMonth());
+    const isPreviousMonthDisabled = minDate
+      && renderDate.getFullYear() === minDate.getFullYear()
+      && minDate.getMonth() >= renderDate.getMonth();
+
+    const isNextMonthDisabled = maxDate
+      && renderDate.getFullYear() === maxDate.getFullYear()
```

Comment:
Following my previous comment, what do you think of changing it to `&& renderDate.getFullYear() >= maxDate.getFullYear()`?